### ISSUE TITLE
When creating a table, ad loading sign to the centre of screen, flip to logs, and make the rest of the screen inactive and non-clickable.

### ISSUE THREAD

**Issue by nairabhishek73** [DalgoT4D/webapp#621]
https://jam.dev/c/e450518d-eb76-429a-bfa7-7cecadceb7b4

Can request Vrushabh to give designs for this. Please share your thoughts/comments

**Comment by fatchat** [comment]
this is very nice but if not part of the original designs will need to be reprioritized

**Comment by nairabhishek73** [comment]
Could we put this at the top of our list of things to do once we achieve everything that's in the designs. Infact there are a number of things that might not match the designs that do exist and I would prioritise this over some of them.

At the very least we need a loading sign.

**Comment by Ishankoradia** [comment]
-  When creating a table or press run button. Grey out the canvas and project tree 
- Switch the tab to logs
- Show spinner till the first line of logs
- logs will populate
- once the run is finished; remove the greyed out lock from canvas + project tree.

**Comment by fatchat** [comment]
@nairabhishek73 released and ready for review

**Comment by nairabhishek73** [comment]
Yes works
